### PR TITLE
fix: fix icon path; fix #100

### LINKED ISSUE #100: app_icon.png not found when run the packaged systeminfo app
When I run `npm run bundle`, First I got a error about 
```
D:\code\other\carlo\examples\systeminfo>pkg package.json
> pkg@4.3.4
> Targets not specified. Assuming:
  node11-linux-x64, node11-macos-x64, node11-win-x64
> Error! No available node version satisfies 'node11'
```

And I found a related issue on https://github.com/zeit/pkg/issues/568,
So I changed the pkg options to use a specified `--targets node10-win-x64` then it packaged successfully.

However when I run the executable generated by `pkg package.json --targets node10-win-x64`, I got the following errors when startup process.

image

### PR DESCRIPTION
**Pull request by geekplux**
Wrap a try-catch for readfile 🤣

### CODE REVIEW

File: lib/carlo.js [hunk 1]
```diff
@@ -27,7 +27,14 @@ const fs = require('fs');
 const util = require('util');
 const {URL} = require('url');
 const EventEmitter = require('events');
-const fsReadFile = util.promisify(fs.readFile);
+const readfile = util.promisify(fs.readFile);
+const fsReadFile = async filename => {
```
Comment (pavelfeldman) [inline lib/carlo.js]: Why is this one needed? You seem to be catching and re-throwing exception right away.

Comment (geekplux) [reply lib/carlo.js]: just because there is an 'unhandled rejection'... I can drop it if you think it's unnecessary.

Comment (pavelfeldman) [reply lib/carlo.js]: Yeah, I think it does not change anything, so we should drop it.

Comment (geekplux) [reply lib/carlo.js]: Done.

**Pull request merged**